src/components/FeatureViews/Accounts/TabComponents/LoadingCourseError.js

```diff
@@ -0,0 +1,13 @@
+import { connect, useSelector } from 'react-redux';
+import React from 'react';
+import loadingCoursesError from "./loadingCoursesError.png";
+
+function LoadingCoursesError(props){
```

Comment:
If you only have 1 prop you want from the user then just destructure the prop to only take the 1 prop you're expecting.
```
function LoadingCoursesError({error})
```